core/primitives/src/bandwidth_scheduler.rs

```diff
@@ -73,8 +73,63 @@ pub struct BandwidthRequestsV1 {
     ProtocolSchema,
 )]
 pub struct BandwidthRequest {
+    /// Requesting bandwidth to this shard.
     pub to_shard: u8,
-    // TODO(bandwidth_scheduler) - store requested bandwidth values inside the BandwidthRequest
+    /// Bitmap which describes what values of bandwidth are requested.
+    pub requested_values_bitmap: BandwidthRequestBitmap,
+}
+
+/// There are this many predefined values of bandwidth that can be requested in a BandwidthRequest.
+pub const BANDWIDTH_REQUEST_VALUES_NUM: usize = 40;
```

Comment:
Sanity check - is versioning supported?

Reply:
`BandwidthRequests` is an enum, so it'll be possible to move to other format for requests in the future.
I didn't want to make `BandwidthRequest` versioned to reduce the serialized size.  Instead of versioning each request we can version the whole list of requests.